components/channel_members_rhs/channel_members_rhs.tsx

```diff
@@ -234,6 +241,30 @@ export default function ChannelMembersRHS({
                 }}
             />
 
+            {/* Users with user management permissions have special restrictions in the default channel */}
+            {(editing && isDefaultChannel && !currentUserIsChannelAdmin) && (
+                <AlertContainer>
+                    <AlertBanner
+                        mode='info'
+                        variant='app'
+                        message={formatMessage({
+                            id: 'channel_members_rhs.default_channel_moderation_restrictions',
+                            defaultMessage: 'In this channel, you can only remove guests. Only <link>channel admins</link> can manage other members.',
+                        }, {
+                            link: (msg: React.ReactNode) => (
+                                <a
+                                    href='https://docs.mattermost.com/welcome/about-user-roles.html#channel-admin'
```

Comment:
Could you please add this link to the docs links here https://github.com/mattermost/mattermost-webapp/blob/master/utils/constants.tsx#L929 ?